src/hooks/useIntersectionObserver.jsx

```diff
@@ -0,0 +1,19 @@
+import { useCallback, useEffect, useState } from 'react'
+
+const useIntersectionObserver = (observerOptions, handleIntersection) => {
+  const [ref, setRef] = useState(null)
+  const handleRef = useCallback(node => {
+    setRef(node)
+  }, [])
+
+  useEffect(() => {
+    if (!ref) {
+      return
+    }
+    const o = new IntersectionObserver(handleIntersection, observerOptions)
```

Comment:
Does Banks already includes the polyfill? 

IntersectionObserver are only available on iOS 12.1+ (https://developer.mozilla.org/fr/docs/Web/API/IntersectionObserver). Banks supports iOS 11, EPA 12.0

Reply:
Thanks, I forgot about the iOS 11 limit. Done in https://github.com/cozy/cozy-banks/pull/2107/commits/b8c57aab9f369201839def45946091be68be794c